### DISCUSSION TITLE
Changing image positioning (paddings)

### DISCUSSION THREAD

**Discussion by sasan-soroush** [LeoNatan/LNPopupController#458] [General] [answered]
How can I change the Popup bar's image positioning? 
I need to set the padding to zero.
something like the attached image

Thanks in advance

![Screen Shot 2021-06-20 at 7 12 39 AM](https://user-images.githubusercontent.com/31206210/122671940-edcf0880-d196-11eb-925d-7aef37be0d47.png)

**Comment by sasan-soroush** [comment, marked as answer]
I actually found the same [issue](https://github.com/LeoNatan/LNPopupController/issues/294) on this repo and I think the way to achieve this view is to create my own LNPopupCustomBarViewController. am I right? or there is a more direct approach to this ?

**Reply by LeoNatan** [reply]
Hello,

Yes, you are right. The layout of the system popup bars is as provided and there is no way to achieve what you need. The best way is to create a custom `LNPopupCustomBarViewController`, where you can put the elements as you wish.
